maint/gdb/gdbhelpers

```diff
@@ -1,9 +1,8 @@
-# Copyright 2018 the Pacemaker project contributors
+# Copyright 2018-2019 the Pacemaker project contributors
 #
 # The version control history for this file may have further details.
-# Author: Jan Pokorny <jpokorny@redhat.com>
-# Part of pacemaker project
-# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# SPDX-License-Identifier: FSFAP
```

Comment:
I'm fine with using FSFAP, but remind me, why is it "substantially more reasonable"? What would be the circumstances where GPLv2+ or LGPLv2.1+ would be too restrictive for these files?